plugin-server/src/cdp/utils/hog-function-filtering.ts

```diff
@@ -320,7 +342,21 @@ export async function filterFunctionInstrumented(options: {
         metrics,
     }
 
+    let preFilterMatch = null
+
     try {
+        // If there are no filters (only bytecode exists then on the filter object)
+        // everything matches no need to execute bytecode (lets save those cpu cycles)
+        if (filters && Object.keys(filters).length === 1 && 'bytecode' in filters) {
+            // if we have no filters we assume we match all events hence match = true
+            hogFunctionPreFilterCounter.inc({ result: 'no_filters' })
+        }
+
+        // check whether we have a match with our pre-filter
+        if (filters?.events?.length) {
```

Comment:
Nit - this could just be in the `preFilterResult` code with the result being an enum of `unknown | match | no-match`